.github/workflows/podvm.yaml

```diff
@@ -40,6 +44,19 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.caa_src_ref }}
+
+    - name: Read properties from versions.yaml
+      run: |
+        # There reference to CAA sources will honored if passed to this
+        # workflow via inputs.
```

Comment:
I'm not native but this sentence is quite hard for me to digest. How about `Allow to override the CAA version from versions.yaml from caa_src_ref input`? (again, not I'm native and not well familiar with CAA)

Reply:
Re-reading what I wrote, I didn't even understood myself :D

Gonna accept your suggestion.